### PR TITLE
[E2E] Tests with Gears and Bloom jsons are added

### PR DESCRIPTION
**Pull request by elenaNaboko**

### CODE REVIEW

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts [hunk 1]
```diff
@@ -167,3 +164,63 @@ test('Verify that user can see that Command Helper displays the previous informa
     //Verify Command helper information
     await t.expect(cliPage.cliHelperTitleArgs.textContent).contains(commandForCheck, 'Command Helper information persists after reopening');
 });
+test.only('Verify that user can work with Gears group in Command Helper (RedisGears module)', async t => {
+    const commandForCheck = 'RG.GETEXECUTION';
+    const commandArgumentsCheck = 'RG.GETEXECUTION id [SHARD|CLUSTER]';
+    const readMoreLink = '#rggetexecution';
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts:170]: 1. I'm not saying it's bad, but you created here variables while you use with only one time within the test.
2. The two first are already defined on top, so you could use with `let` in order to re-use here.

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts [hunk 2]
```diff
@@ -167,3 +164,63 @@ test('Verify that user can see that Command Helper displays the previous informa
...
+        'CF.ADD'
+    ];
+    const commandArgumentsCheck = [
+        'BF.MEXISTS key item [item ...]',
+        'CMS.QUERY key item [item ...]',
+        'TDIGEST.RESET key',
+        'TOPK.LIST key numKeys WITHCOUNT',
+        'CF.ADD key item'
+    ];
+    const externalPage = [
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts:201]: You could use `let` on top and re-define here.

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts [hunk 3]
```diff
@@ -167,3 +164,63 @@ test('Verify that user can see that Command Helper displays the previous informa
     //Verify Command helper information
     await t.expect(cliPage.cliHelperTitleArgs.textContent).contains(commandForCheck, 'Command Helper information persists after reopening');
 });
+test.only('Verify that user can work with Gears group in Command Helper (RedisGears module)', async t => {
```
[the remark this answers is not in the document]
Comment (elenaNaboko) [reply tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts]: Removed

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts (hunk 1 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts]: Thanks! I re-worked all tests from this file

File: tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts (hunk 2 above)
Comment (elenaNaboko) [reply tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts]: Fixed

**Review by ofersteinberg — APPROVED** [review]
I see the fixture contains too many tests, consider splitting it in the future.

**Review by tanyatrayanava — APPROVED** [review]

**Pull request merged**